app-backend/database/src/main/scala/com/azavea/rf/database/tables/Projects.scala

```diff
@@ -86,6 +86,42 @@ object Projects extends TableQuery(tag => new Projects(tag)) with LazyLogging {
     }
   }
 
+  /** Get the AOIs belonging to a project. */
+  def listAOIs(
+    projectId: UUID,
+    pageRequest: PageRequest,
+    user: User
+  )(implicit database: DB): Future[PaginatedResponse[AOI]] = {
+    // TODO: This should be a join?
```

Comment:
Scanning through the code base for examples, it wasn't clear how this two-step process should be done. I found examples of `Foo.WithRelated` that seemed to be the marshalled types for joins, but the added boilerplate turned me off. I ended up making two calls here and `flatMap`ping the Futures.

PROs:
- Less boilerplate, result marshals straight to an `AOI`

NEUTRAL:
- The first filter will only ever match on one Project, so handling the fallout of a join felt like overkill

CON:
- This makes two DB calls instead of one